streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -313,36 +282,29 @@ public void shouldNotFlushAndThrowOnCloseDirty() {
 
         final double expectedCloseTaskMetric = 1.0;
         verifyCloseTaskMetric(expectedCloseTaskMetric, streamsMetrics, metricName);
-
-        EasyMock.verify(stateManager);
     }
 
     @Test
     public void shouldNotThrowFromStateManagerCloseInCloseDirty() {
-        EasyMock.expect(stateManager.changelogOffsets()).andStubReturn(Collections.emptyMap());
-        stateManager.close();
-        EasyMock.expectLastCall().andThrow(new RuntimeException("KABOOM!")).anyTimes();
-        EasyMock.replay(stateManager);
+        doThrow(new RuntimeException("KABOOM!")).when(stateManager).close();
 
         task = createStandbyTask();
         task.initializeIfNeeded();
 
         task.suspend();
         task.closeDirty();
 
-        EasyMock.verify(stateManager);
+        verify(stateManager, atLeastOnce()).close();
     }
 
     @Test
     public void shouldSuspendAndCommitBeforeCloseClean() {
         stateManager.close();
-        EasyMock.expectLastCall();
+        verify(stateManager, atLeastOnce()).close();
         stateManager.checkpoint();
-        EasyMock.expectLastCall().once();
-        EasyMock.expect(stateManager.changelogOffsets())
-                .andReturn(Collections.singletonMap(partition, 60L));
-        EasyMock.expect(stateManager.changelogPartitions()).andReturn(Collections.singleton(partition)).anyTimes();
```

Comment:
You did not migrate this line neither to a stubbing nor to a verification.

Reply:
This was another case where I didn't know what `EasyMock.verify(stateManager)` at the end of the method was actually verifying.

I placed 

```java
when(stateManager.changelogOffsets())
    .thenReturn(Collections.singletonMap(partition, 60L));
when(stateManager.changelogPartitions())
        .thenReturn(Collections.singleton(partition));
```

back and verified `verify(stateManager).changelogOffsets();`